bigbluebutton-html5/imports/ui/components/video-provider/service.js

```diff
@@ -577,12 +581,19 @@ class VideoService {
     return isLocal ? 'share' : 'viewer';
   }
 
-  getSkipVideoPreview(fromInterface = false) {
+  getSkipVideoPreview() {
+    if (this.skipVideoPreviewOnFirstJoin === null) {
+      this.skipVideoPreviewOnFirstJoin = getFromUserSettings('bbb_skip_video_preview_on_first_join', false) || SKIP_VIDEO_PREVIEW_ON_FIRST_JOIN;
+    }
     if (this.skipVideoPreview === null) {
       this.skipVideoPreview = getFromUserSettings('bbb_skip_video_preview', false) || SKIP_VIDEO_PREVIEW;
     }
 
-    return this.skipVideoPreview && !fromInterface;
+    if(this.isFirstJoin){
+      return this.skipVideoPreviewOnFirstJoin || this.skipVideoPreview;
```

Comment:
You could simplify this whole block to
`return (this.isFirstJoin && this.skipVideoPreviewOnFirstJoin) || this.skipVideoPreview;

Reply:
Also please lint your new code (only)